src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml

```diff
@@ -348,847 +359,978 @@
+                                                               ButtonId="InvSech"
+                                                               Click="FlyoutButton_Clicked"
+                                                               Content="sech⁻¹"
+                                                               IsTabStop="false"
+                                                               IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+
+                                    <controls:CalculatorButton x:Name="InvcschButton"
+                                                               Grid.Row="1"
+                                                               Grid.Column="1"
+                                                               Style="{StaticResource OperatorButtonStyle}"
+                                                               AutomationProperties.AutomationId="invcschButton"
+                                                               ButtonId="InvCsch"
+                                                               Click="FlyoutButton_Clicked"
+                                                               Content="csch⁻¹"
+                                                               IsTabStop="false"
+                                                               IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+
+                                    <controls:CalculatorButton x:Name="InvcothButton"
+                                                               Grid.Row="1"
+                                                               Grid.Column="2"
+                                                               Style="{StaticResource OperatorButtonStyle}"
+                                                               AutomationProperties.AutomationId="invcothButton"
+                                                               ButtonId="InvCoth"
+                                                               Click="FlyoutButton_Clicked"
+                                                               Content="coth⁻¹"
+                                                               IsTabStop="false"
+                                                               IsEnabled="{x:Bind TrigButton.IsEnabled, Mode=OneWay}"/>
+                                </Grid>
                             </Grid>
-
-                            <Grid x:Name="InverseTrigFunctions"
-                                  Grid.RowSpan="2"
-                                  Grid.Column="1"
-                                  Grid.ColumnSpan="3"
-                                  Visibility="Collapsed">
+                        </Flyout>
+                    </controls:OperatorPanelButton.FlyoutMenu>
+                </controls:OperatorPanelButton>
+
+                <controls:OperatorPanelButton x:Name="InequalityButton"
```

Comment:
Equation gets submitted incorrectly in this scenario:

1. Click on the first equation box, click '2', click '5', click the submit button
2. Equation submits with a line on 25, focus goes to the new ghost text box
3. Now that focus is automatically on the ghost text box, type '2', type '5', click the inequality flyout button
4. Equation incorrectly submits after clicking the inequality flyout button, flyout never opened

Reply:
I'm not quite sure how to solve this.  When tracking focus in the debugger,  focus is not moving.  However the loss of focus event is getting fired.

Reply:
I will take a look at this tomorrow morning to see if I can help track it down

Reply:
Turns out there is a bug in Xaml.  Worked around this in code.